src/pages/_document.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * This file is needed to embed MUI theme CSS into the pre-built HTML files
+ * @see https://github.com/mui/material-ui/tree/master/examples/nextjs-with-typescript
+ */
+import Document, { Html, Head, Main, NextScript, DocumentContext } from 'next/document'
+import createEmotionServer from '@emotion/server/create-instance'
+import createEmotionCache from '@/utils/createEmotionCache'
+
+export default class WebCoreDocument extends Document {
+  render() {
+    return (
+      <Html lang="en">
+        <Head>
+          <meta name="emotion-insertion-point" content="" />
```

Comment:
```suggestion
        <Head>
          <title>Safe</title>
          <meta name="emotion-insertion-point" content="" />
```
Getting a warning that `title` is required but missing.

Reply:
Removed it again, it shows a lint error with it.